frontend/lib/features/audio-player/player_button.widget.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:just_audio/just_audio.dart';
+import 'package:mbl/themes/themes.dart';
+
+Widget playerButton(PlayerState? playerState, AudioPlayer player) {
+  final processingState = playerState?.processingState;
+  if  (processingState == ProcessingState.loading ||
```

Comment:
I think it looks uncommon to have the player controls loading. Instead I would rather disable their functionality(for example with an if) until the player has fully loaded